src/main/g8/README.md

```diff
@@ -89,19 +89,28 @@ Below are the libraries used to provide a broad starting base for this project.
 
 
 
-## [sbt-native-packager](https://github.com/sbt/sbt-native-packager) - JRE-less executable ##
+## [sbt-native-image](https://github.com/scalameta/sbt-native-image) - JRE-less executable ##
 
-`sbt-native-packager` is an sbt plugin that allows sbt to build various distributables.
+`sbt-native-image` is an sbt plugin that allows sbt to build assembled JRE-less executables using GraalVM.  This plugin handles downloading and installation of the related Graal JDK and native-image tool which are required to make the executable binary.  However, these are very particular about their targets - That's where we pick up with docker in this repo, ultimately ending up with a portable, fast, minified docker image with a single binary in it:
+
+1. **sbt + sbt-native-image + GraalVM + native-image** get us from **scala-to-bytecode-to-llvm-to-libc**.
+2. **docker** gets us from **libc-to-kernel**, providing a consistent "target" for GraalVM & native-image.
+
+However, it isn't all roses.  **The native-image & AoT aspect of any project building with Graal is very brittle**, as all classes exposed exclusively via reflection at run-time [must be explicitly provided to Graal](https://www.graalvm.org/reference-manual/native-image/Reflection/), whether determined automatically or manually.  
+
+Do your reading before you decide to push the outputs of native-image to production, and consider the cost of maintaining reflection mappings.
```

Comment:
> consider the cost of maintaining reflection mappings

If it were only reflection mappings I'd be stoked. For me, most often it's the substitutions and random LLVM shit that really grinds my goats (getting segfaults because dns lookups require dynamically linked libc [for example](https://github.com/oracle/graal/issues/571)). 

I'm only mentioning it because (assuming you're crazy enough to use it, I always advice folks to avoid it) graal native's tracing agent does as pretty dang good job of making all your reflection problems disappear. So if you give folks the impression that reflection is the only issue, they might get a false sense of security. Though, I suppose if they were to truly heed your warning at the start of that sentence ("Do your reading"), they might figure that out for themselves.

That all being said, I ain't saying I know how to make an effective warning sign for graal headaches either. I think [the disclaimer](https://carpe.github.io/scalambda/docs/thegraaaaal/#disclaimers) I used for Scalambda was far too wordy. So much so that I wouldn't be surprised if anyone has ever actually read it.

Reply:
I'm gonna add those comments to this README. Maybe it saves someone.